package.json

```diff
@@ -14,9 +14,12 @@
         "@types/node": "^12.0.0",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
+        "@types/react-draft-wysiwyg": "^1.13.2",
```

Comment:
Please install this as a devDependency

Reply:
![image](https://user-images.githubusercontent.com/58205909/122312133-cc271480-cf3d-11eb-91d3-14064a13a2d9.png)
I done it